src/deepsparse/server/config.py

```diff
@@ -144,16 +149,36 @@ class ServerConfig(BaseModel):
 
     endpoints: List[EndpointConfig] = Field(description="The models to serve.")
 
-    loggers: Union[Dict[str, Dict[str, Any]], str, None] = Field(
+    loggers: Optional[List[Union[str, Dict[str, Dict[str, Any]]]]] = Field(
```

Comment:
```suggestion
    loggers: Dict[str, Dict[str, Any]]
```

Reply:
Ok, this makes sense, I also like the simplification, but why removing the `Optional`?

Reply:
I was thinking empty dict is equivalent to `None`, and it would make typing a bit easier. Fine with keeping optional though

Reply:
Actually, I still like the typing (well, I do not like how complicated it is but it expresses the right idea).
In the worst-case (most complex) scenario, the user could have the following setting:

```yaml
loggers:
    - python
    - some_other_logger
    - prometheus:
        - arg1: value
        - arg2: value
    - some_other_logger_1:
        - arg1: value
        - arg2: value
```

To me, such representation feels quite intuitive, and would like to support it in the `ServerConfig`.
It brakes down to a list that contains two strings (`['python', 'some_other_logger']) and two dictionaries
(['prometheus: {...}, `some_other_logger_1' :{...}']). @bfineran @corey-nm what am I missing here?

Reply:
(this is why I think that typing: `Optional[List[str, Dict[str, Dict[str, Any]]]]` still makes sense.

Reply:
can that not just be represented by a dict?

```yaml
loggers:
    python: {}
    some_other_logger: {}
    prometheus:
        - arg1: value
        - arg2: value
    some_other_logger_1:
        - arg1: value
        - arg2: value
```

I think dict is better repesentation here because names of loggers are unique. E.g. we can't have
```yaml
loggers:
  - prometheus: { ... }
  - prometheus: { ... }
```

Reply:
as mentioned in comments in `build_logger` agreed on just `dict`, we want to enforce simple, consistent behavior

Reply:
As mentioned in a different comment of mine, `python: {}` feels weird to me.